cfn-resources/util/progressevent/failed_event.go

```diff
@@ -29,7 +29,7 @@ func GetFailedEventByResponse(message string, response *http.Response) handler.P
 		HandlerErrorCode: getHandlerErrorCode(response)}
 }
 
-func GetFailedEventByCode(message, handlerErrorCode string) handler.ProgressEvent {
+func GetFailedEventByCode(message string, handlerErrorCode string) handler.ProgressEvent {
```

Comment:
```suggestion
func GetFailedEventByCode(message, handlerErrorCode string) handler.ProgressEvent {
```

String is not needed here